### PR TITLE
DNN-10279 Issue verification and code refactoring

### PR DESCRIPTION
**Pull request by mean2me**

### CODE REVIEW

File: DNN Platform/Modules/Journal/ServicesController.cs [hunk 1]
```diff
@@ -392,48 +347,7 @@ public HttpResponseMessage PreviewUrl(PreviewDTO postData)
...
-//                message = string.Format(Localization.GetString("SaveFileError"), sourceFile.FileName);
-//            }
-//            if (String.IsNullOrEmpty(message) && fi != null)
-//            {
-//                return Json(fi);
-//            }
-//
-//            return Json(new { Result = message });
-//        }
-
```
Comment (tpluscode) [inline DNN Platform/Modules/Journal/ServicesController.cs]: 👍 to removing dead code

File: DNN Platform/Modules/Journal/ServicesController.cs [hunk 2]
```diff
@@ -98,135 +98,157 @@ private static bool IsAllowedLink(string url)
...
+                if (postData.ProfileId == -1)
+                {
+                    postData.ProfileId = UserID;
                 }
 
+                checkProfileAccess(postData.ProfileId, UserInfo);
+
+                checkGroupAccess(ref postData);
+
+                // JI
```
Comment (tpluscode) [inline DNN Platform/Modules/Journal/ServicesController.cs]: c'mon, why is this comment here? ;)

File: DNN Platform/Modules/Journal/ServicesController.cs [hunk 3]
```diff
@@ -98,135 +98,157 @@ private static bool IsAllowedLink(string url)
...
+                var ji = prepareJournalItem(ref postData, ref mentionedUsers);
```
Comment (tpluscode) [inline DNN Platform/Modules/Journal/ServicesController.cs]: By the way, while you're cleaning up maybe you could actually rename that `ji` into `journalItem`

Comment (mean2me) [reply DNN Platform/Modules/Journal/ServicesController.cs]: done

File: DNN Platform/Modules/Journal/ServicesController.cs (hunk 2 above)
Comment (mean2me) [reply DNN Platform/Modules/Journal/ServicesController.cs]: LOL :D

File: DNN Platform/Modules/Journal/ServicesController.cs [hunk 4]
```diff
@@ -105,159 +105,161 @@ public HttpResponseMessage Create(CreateDTO postData)
...
-                        journalTypeId = 2;
-                        break;
-                    case "photo":
-                        journalTypeId = 3;
-                        break;
-                    case "file":
-                        journalTypeId = 4;
-                        break;
-                }
+                int UserID = UserInfo.UserID;
```
Comment (tpluscode) [inline DNN Platform/Modules/Journal/ServicesController.cs]: local variables typically start with lower case

**Pull request merged**